src/extensions/default/bramble/lib/RemoteCommandHandler.js

```diff
@@ -99,6 +99,14 @@ define(function (require, exports, module) {
             PreferencesManager.set("allowJavaScript", false);
             PostMessageTransport.reload();
             break;
+        case "BRAMBLE_ENABLE_LINT":
+            PreferencesManager.set("allowLint", true);
```

Comment:
You have to set this for the proper pref in your extension:

```js
PreferencesManager.getExtensionPrefs("interactive-linter").set("enabled" true);
```

I think.  Test that.